src/libs/actions/Policy/Policy.ts

```diff
@@ -2578,7 +2578,10 @@ function createWorkspaceFromIOUPayment(iouReport: OnyxEntry<Report>): WorkspaceF
     });
 
     // We need to move the report preview action from the DM to the workspace chat.
-    const reportPreview = ReportActionsUtils.getParentReportAction(iouReport);
+    const parentReport = ReportAcionsConnection.getAllReportActions()?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${iouReport.parentReportID}`];
```

Comment:
@BrtqKr Not sure if I am missing something here but why did we have to create `ReportAcionsConnection` when we already have a utility `ReportActionsUtils.getAllReportActions`?

Reply:
because we have a new eslint rule, which is forcing use to fix those things when we edit files containing deprecated code

Reply:
@BrtqKr Ah! I see. So, the deprecated function is `getParentReportAction` but is there a need to create `ReportAcionsConnection` when `ReportActionsUtils.getAllReportActions` gives us the same result?

Reply:
This is something I'll need to discuss separately, but from what I understand the main issue is that the rule has been prepared with components in mind, so it's expecting some kind of action to be taken anyway. I was thinking about the possible benefits of having a connection in one place instead of having logic split between different files, but as I said earlier, I'll discuss it in more detail with someone who has more context.

Reply:
This doesn't feel like a blocker.

Reply:
> This is something I'll need to discuss separately

`getParentReportAction` has been used in many places and I agree that this needs a separate discussion. Since this is not at all related to this issue, I would prefer not to include the changes for this in this PR. As and when there is a conclusion on the way forward, we can add the code changes. Anyway, I would leave the final decision on this to @marcaaron. 

If we are leaving the code as it is, let us correct the spelling `ReportAcionsConnection`.